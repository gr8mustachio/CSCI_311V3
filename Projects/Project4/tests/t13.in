5
addEdge 0 1
addEdge 0 2 
addEdge 0 3
addEdge 1 2
addEdge 3 4
printGraph
twoColoring
