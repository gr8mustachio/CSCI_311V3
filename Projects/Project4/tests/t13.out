Adjacency lists of the graph are:
1 2 3 
0 2 
0 1 
0 4 
3 
Two-coloring is NOT possible in this graph.
